AutoCollection/CorrelationContextManager.ts

```diff
@@ -97,6 +97,13 @@ export class CorrelationContextManager {
         }
     }
 
+    /**
+     * Wrapper for cls-hooked bindEmitter method
+     */
+    public static bindEmitter(reqres: any): void {
```

Comment:
Should we name this `wrapEmitter` to be consistent with `wrapCallback`? Also, shouldn't the type here be `EventEmitter`?

Reply:
Minor nit: Would prefer a generic argument name like `emitter` to `reqres`

Reply:
This also needs if guards to only act when this is enabled